pkg/component/controlplane/control_plane.go

```diff
@@ -0,0 +1,64 @@
+package controlplane
+
+import (
+	"fmt"
+	"strings"
+
+	"istio.io/operator/pkg/apis/istio/v1alpha2"
+	"istio.io/operator/pkg/component/feature"
+	"istio.io/operator/pkg/translate"
+	"istio.io/operator/pkg/util"
+)
+
+// IstioControlPlane is an installation of an Istio control plane.
+type IstioControlPlane struct {
+	features []feature.IstioFeature
+	started  bool
+}
+
+// NewIstioControlPlane creates a new IstioControlPlane and returns a pointer to it.
+func NewIstioControlPlane(installSpec *installerv1alpha1.IstioControlPlaneSpec, translator *translate.Translator) *IstioControlPlane {
+	opts := &feature.FeatureOptions{
+		InstallSpec: installSpec,
+		Traslator:   translator,
+	}
+	return &IstioControlPlane{
+		features: []feature.IstioFeature{
+			feature.NewTrafficManagementFeature(opts),
+			feature.NewSecurityFeature(opts),
+			feature.NewPolicyFeature(opts),
+			feature.NewTelemetryFeature(opts),
+			feature.NewConfigManagementFeature(opts),
+			feature.NewAutoInjectionFeature(opts),
+		},
+	}
+}
+
+// Run starts the Istio control plane.
+func (i *IstioControlPlane) Run() error {
```

Comment:
Features are part of the the IstioControlPlane, which is responsible for starting them all. Basically, Run and Render and designed to be recursive, so that each Feature is also responsible for calling the functions of all its constituent components.